src/qibolab/platform.py

```diff
@@ -170,6 +169,283 @@ def create_tii_qw5q_gold(runcard, simulation_duration=None, address=None, cloud=
     return platform
 
 
+def create_tii_IQM5q(runcard, descriptor=None):
+    """Create platform using Zurich Instrumetns (Zh) SHFQC, HDAWGs and PQSC.
+
+    Instrument related parameters are hardcoded in ``__init__`` and ``setup``.
+
+    Args:
+        runcard (str): Path to the runcard file.
+        descriptor (str): Instrument setup descriptor.
+            If ``None`` it will attempt to connect to TII whole Zurich instruments setup.
+    """
+    # Create channel objects
+    channels = ChannelMap()
+    # readout
+    channels |= ChannelMap.from_names("L3-31")
```

Comment:
We should extend this operator (`__ixor__`) to avoid repeating these `ChannelMap.from_names()` calls.

This I will do by myself, and it is not going to break anything :)

Reply:
@AleCandido It didnt work, some got saved character per character I think

ChannelMap(_channels={'L': Channel(name='L', ports=[], _local_oscillator=None, power_range=None, _bias=None, _filter=None, max_bias=None), '3': Channel(name='3', ports=[], _local_oscillator=None, power_range=None, _bias=None, _filter=None, max_bias=None), '-': Channel(name='-', ports=[], _local_oscillator=None, power_range=None, _bias=None, _filter=None, max_bias=None), '1': Channel(name='1', ports=[], _local_oscillator=None, power_range=None, _bias=None, _filter=None, max_bias=None)

or something weird at least for the L3-31 line